### ISSUE TITLE
Not Issues.  Why is "Weight decay" parameter so large?

### ISSUE THREAD

**Issue by huhubrave** [mlfoundations/open_clip#32]
Question, not Issues.

Why is Weight decay so large?  (The default   0.2 or 0.1)
Usually is 1e4

**Comment by mitchellnw** [comment]
Thanks for the questions -- that is also the value mentioned in the the CLIP paper, and is also close to the default AdamW weight decay https://pytorch.org/docs/stable/generated/torch.optim.AdamW.html. I believe values ~ 1e-4 are used for SGD

**Comment by carlini** [comment]
You also have to be careful to distinguish between when people say weight decay meaning (weight decay * learning rate) or just straight weight decay to the parameters not scaled by learning rate.

**Comment by huhubrave** [comment]
Thank you for your reply. I get it, I will close this question
